src/Trakx.CryptoCompare.ApiClient.Websocket.Tests/Unit/CryptoCompareWebsocketHandlerTests.cs

```diff
@@ -75,4 +114,10 @@ public async Task IncomingMessage_should_map_to_correct_observer()
         topicMessage!.Symbol.Should().Be(fullVolumeMsg.Symbol);
         topicMessage!.Volume.Should().Be(fullVolumeMsg.Volume);
     }
+
+    private static TopicSubscription SetupTopicSubscription(SubscribeActions action)
+    {
+        var subscriptionString = CryptoCompareSubscriptionFactory.GetFullTopTierVolumeSubscriptionStr("test");
```

Comment:
[suggestion] maybe use a more meaningful string then 'test'

Reply:
"test" comes from the previous version. Would "testSymbol" work here?

Reply:
Done via const.